OracleTuxedo/salt/samples/bankapp/README.md

```diff
@@ -0,0 +1,54 @@
+
+# Oracle Tuxedo `bankapp` Sample Container Image
+
+This sample extends the Oracle Tuxedo image by creating a "bankapp" application.
+
+## Prerequisites
+
+This image uses the Orace Tuxedo container image `oracle/tuxedo:latest` as its base. Please follow the [Oracle Tuxedo image](https://github.com/oracle/docker-images/tree/main/OracleTuxedo/core) documentation to build the container image before performing the operations described in this document.
+
+## How to Build the Image
+
+Download the parent directory (bankapp) of this README.md file from GitHub to a location of your choice. This location is referenced as `YOUR_APP_DIR` in this document.
+
+```shell
+export TUXDIR=<Tuxedo installation directory when the above mentioned `oracle/tuxedo:latest` image was built>
+cp -rp ${TUXDIR}/samples/atmi/bankapp/*  &lt;YOUR_APP_DIR>/bankapp/
```

Comment:
```suggestion
cp -rp ${TUXDIR}/samples/atmi/bankapp/*  "$YOUR_APP_DIR/bankapp/"
```

Reply:
Please don't use < or > in sample commands as they won't work if cut and paste. Your commands in code blocks should work.

Reply:
@Djelibeybi is that really a feasible mandate for samples in general e.g. if schema credentials are needed?

Reply:
If schema credentials are required, you could document how to set those as variables and then use the variables for the sample commands. So yeah, I think it's feasible.